packages/11ty/_plugins/figures/tests/__fixtures__/figures/zoomable/manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "@context": "http://iiif.io/api/presentation/3/context.json",
+  "id": "http://localhost:8080/iiif/zoom/manifest.json",
+  "type": "Manifest",
+  "behavior": [
+    [
+      "continuous",
+      "sequence"
+    ]
+  ],
```

Comment:
@anderspollack unless I am mistaken the `sequence` value for [behavior](https://iiif.io/api/presentation/3.0/#behavior) is only 
> valid only on Ranges, where the Range is [referenced](https://iiif.io/api/presentation/3.0/#12-terminology) in the structures property of a Manifest.
Interestingly, the following part of the the spec provides guidance on how a figure with multiple sequences might indicate the UI to _page_ through the sequences that are each `continous` --
> A Range with the behavior value continuous, in a Manifest with the behavior value paged, would mean that the Manifest’s Canvases should be rendered in a paged fashion, unless the range is selected to be viewed, and its included Canvases would be rendered in that context only as being virtually stitched together.